content/en-us/scripting/security/security-tactics.md

```diff
@@ -68,20 +68,29 @@ local newPart = remoteFunction:InvokeServer(Color3.fromRGB(200, 0, 50), Vector3.
 
 if newPart then
 	print("The server created the requested part:", newPart)
+elseif newPart == false then
+	print("The server denied the request. No part was created")
 end
 ```
 
-```lua title="Script in ServerScriptService" highlight="4, 8-10, 12, 19"
+```lua title="Script in ServerScriptService" highlight="4, 6-7, 10-17, 19, 28"
 local ReplicatedStorage = game:GetService("ReplicatedStorage")
 
 local remoteFunction = ReplicatedStorage:WaitForChild("RemoteFunctionTest")
 local t = require(ReplicatedStorage:WaitForChild("t"))
 
+-- Create type validator in advance to avoid unnecessary overhead
+local createPartTypeValidator = t.tuple(t.instanceIsA("Player"), t.Color3, t.Vector3)
```

Comment:
```suggestion
-- Create type validator in advance to avoid unnecessary overhead
local createPartTypeValidator = t.tuple(t.instanceIsA("Player"), t.Color3, t.Vector3)

```